client/app/services/authentication/authentication.service.js

```diff
@@ -21,6 +21,12 @@ class AuthenticationService {
       .catch((res) => this.$q.reject(res.data));
   }
 
+  verify(userEmail) {
```

Comment:
This should be renamed appropriately, to `resendVerificationRequest` or similar.